Code/Engine/Foundation/IO/FileSystem/Implementation/FileSystem.cpp

```diff
@@ -502,15 +502,18 @@ ezDataDirectoryReader* ezFileSystem::GetFileReader(ezStringView sFile, ezFileSha
     // Let the data directory try to open the file.
     ezDataDirectoryReader* pReader = s_pData->m_DataDirectories[i].m_pDataDirType->OpenFileToRead(sRelPath, FileShareMode, bOneSpecificDataDir);
 
-    if (bAllowFileEvents && pReader != nullptr)
+    if (pReader != nullptr)
```

Comment:
Same issue with the GetFileWriter method below

Reply:
Thanks! Fixed.